libs/sdk-ui-kit/src/Icon/icons/UserGroup.tsx

```diff
@@ -0,0 +1,26 @@
+// (C) 2021-2024 GoodData Corporation
+import React from "react";
+
+import { IIconProps } from "../typings.js";
+
+/**
+ * @internal
+ */
+export const UserGroup: React.FC<IIconProps> = ({ color, className, width, height }) => {
+    return (
+        <svg
+            className={className}
+            width={width ?? 27}
+            height={height ?? 27}
+            viewBox="0 0 27 27"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <rect width="27" height="27" rx="13.5" fill="#EBEFF4" />
```

Comment:
I think we want another color variable for the fill here. Check other icons, I believe some of them have multiple colors.